eth-reference-tests/src/referenceTest/resources/fork_choice_tests/cache/attestation_0.yaml

```diff
@@ -0,0 +1,12 @@
+aggregation_bits: '0x11'
```

Comment:
Looks like you've added these files to `fork-choice-tests` resources and `eth-reference-tests` resources.  Should these ref-test files be deleted?

Reply:
Yep, the ones in eth-reference-tests should be deleted. Good catch.

Reply:
Done.